docs/source/user-guide/ui-api-guide/external-datasets/managing.md

```diff
@@ -1,27 +1,36 @@
 # Managing External Datasets
 
-External profiles are quite similar to the profiles produced by a Merlin simulation. They describe the behavior of some dynamic characteristic over time and can be viewed in the Aerie UI along with a plan. External profiles come from the user, rather than Merlin's simulation capabilities, and can be uploaded at any time, before or after simulation.
+External datasets contain profiles that are quite similar to the profiles produced by a Merlin simulation. They describe the behavior of some dynamic characteristic over time and can be viewed in the Aerie UI along with a plan. External profiles come from the user, rather than Merlin's simulation capabilities, and can be uploaded at any time before or after simulation.
```

Comment:
While _we_ know the shape of profiles produced by merlin simulation - I think we should assume that someone coming to this page might not know. We can either include a link to where that format is described, or describe it here.

In particular, I think it's important to define the word `profile`, as well as the concept of a `profile_segment` somewhere.

(May be out of scope of your current effort - we can follow up on this)

Reply:
Happy to add it to this commit. Do we have a good working definition for each?

Reply:
Not sure about a working definition, I'll try to brainstorm something... (feel free to weigh in @joelcourtney) some of this may be redundant with the rest of this page, I'm just spitballing:

A resource profile describes the behavior of a resource over time. Each profile has a name, a type, a start time, and a list of profile segments that conform to that type. 

Each profile segment defines the value of the resource over a duration of time (referred to as its "dynamics"). The first segment starts at the start time of the profile, and every subsequent segment starts at the end of the previous segment.

A segment can also represent a gap in data - in which case it would have a duration but no dynamics.